src/js/classes/AnyOfValidator.ts

```diff
@@ -55,7 +56,11 @@ class AnyOfValidator extends BaseValidator<any> {
             this.onError(value, otherValues);
         }
 
-        return [false, this.defaultErrorMsg];
+        if (t) {
+            return [false, t('any_of') as string];
```

Comment:
I played around with improving the typing a little locally and am reasonably sure we can teach the typing system that this is a string without the need to cast. It is somewhat complex tho and would involve design decisions I don't feel comfortable making on my own (e.g.: do we allow any string as the key parameter of `t` or do we want to restrict it to the keys defined in i18n.js? Yes, there is more nuance to this question.)

Let's get together when you're back again and I can show you potential solves for this and we can discuss which one to go with :)

Reply:
I believe have to allow any string to be given to the `t` function because user defined validators may use different translation keys than what we do

Reply:
If you're available, let's talk about this later today. I have some ideas but it's easier for me to explain them in person.

I wouldn't necessarily consider this a blocker either way tho. It is more type safety than before and if we decide in the future to improve typing to make casts like this unnecessary, I don't think that would break stuff (aka a minor release should be enough, I don't think it would need another major release).